src/iqm_client/iqm_client.py

```diff
@@ -252,14 +249,15 @@ def __init__(
     def submit_circuit(
             self,
             circuit: Circuit,
-            qubit_mapping: list[SingleQubitMapping],
+            qubit_mapping: Optional[list[SingleQubitMapping]] = None,
             shots: int = 1
     ) -> UUID:
         """Submits a quantum circuit to be executed on a quantum computer.
 
         Args:
             circuit: circuit to be executed
-            qubit_mapping: mapping of human-readable qubit names in ``circuit`` to physical qubit names
+            qubit_mapping: mapping of human-readable qubit names in ``circuit`` to physical qubit names,
+                           if circuit uses physical qubit names qubit_mapping can be set to None (default)
```

Comment:
```suggestion
            qubit_mapping: Mapping of human-readable (logical) qubit names in ``circuit`` to physical qubit names.
                Can be set to ``None`` if ``circuit`` already uses physical qubit names.
```
Sphinx finds the default value for each parameter in the signature, normally it's better not to repeat it.

Reply:
fixed as suggested